subsys/bootloader/bl_client/bl_client.c

```diff
@@ -97,3 +111,9 @@ int bl_secp256r1_validate(const u8_t *hash, u32_t hash_len,
 	return get_bl_secp256r1_abi()->abi.bl_secp256r1_validate(hash, hash_len,
 							public_key, signature);
 }
+
+bool bl_validate_firmware(u32_t fw_dst_address, u32_t fw_src_address)
```

Comment:
Where is this function linked in to the bl_client ABI?

Reply:
I'm not entirely sure what you are asking, but as a test you could read the newly written fw_metadata docs and see if you understand it afterwards :).

fw_metadata docs are in this PR: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1291

Reply:
Or comment there what questions it doesn't answer.